content/docs/reference/config/installation-options-version-comparison/index.md

```diff
@@ -0,0 +1,445 @@
+| `gateways.istio-ingressgateway.meshExpansionPorts.targetPort` | `853` | `` |
+| `gateways.istio-ingressgateway.meshExpansionPorts.name` | `tcp-dns-tls` | `` |
+| `gateways.istio-ingressgateway.env.ISTIO_META_ROUTER_MODE` | `"sni-dnat"` | `` |
+| `gateways.istio-ingressgateway.nodeSelector` | `{}` | `` |
+| `gateways.istio-egressgateway.autoscaleEnabled` | `true` | `` |
+| `gateways.istio-egressgateway.cpu.targetAverageUtilization` | `80` | `` |
+| `gateways.istio-egressgateway.podAnnotations` | `{}` | `` |
+| `gateways.istio-egressgateway.ports.targetPort` | `15443` | `` |
+| `gateways.istio-egressgateway.env.ISTIO_META_ROUTER_MODE` | `"sni-dnat"` | `` |
+| `gateways.istio-egressgateway.nodeSelector` | `{}` | `` |
+| `gateways.istio-ilbgateway.autoscaleEnabled` | `true` | `` |
+| `gateways.istio-ilbgateway.cpu.targetAverageUtilization` | `80` | `` |
+| `gateways.istio-ilbgateway.podAnnotations` | `{}` | `` |
+| `gateways.istio-ilbgateway.nodeSelector` | `{}` | `` |
+
+## `kiali` options
+
+| New Key | New Default Value | New Description|
+| --- | --- | --- |
+| `kiali.contextPath` | `/kiali` | `` |
+| `kiali.nodeSelector` | `{}` | `` |
+| `kiali.ingress.hosts` | `kiali.local` | `` |
+| `kiali.dashboard.secretName` | `kiali` | `` |
+| `kiali.dashboard.usernameKey` | `username` | `` |
+| `kiali.dashboard.passphraseKey` | `passphrase` | `` |
+| `kiali.prometheusAddr` | `http://prometheus:9090` | `` |
+| `kiali.createDemoSecret` | `false` | `` |
+
+## `istiocoredns` options
+
+| New Key | New Default Value | New Description|
+| --- | --- | --- |
+| `istiocoredns.enabled` | `false` | `` |
+| `istiocoredns.replicaCount` | `1` | `` |
+| `istiocoredns.coreDNSImage` | `coredns/coredns:1.1.2` | `` |
+| `istiocoredns.coreDNSPluginImage` | `istio/coredns-plugin:0.1-istio-1.1` | `` |
+| `istiocoredns.nodeSelector` | `{}` | `` |
+
+## `security` options
+
+| New Key | New Default Value | New Description|
+| --- | --- | --- |
+| `security.enabled` | `true` | `` |
+| `security.createMeshPolicy` | `true` | `` |
+| `security.nodeSelector` | `{}` | `` |
+
+## `nodeagent` options
+
+| New Key | New Default Value | New Description|
+| --- | --- | --- |
+| `nodeagent.enabled` | `false` | `` |
+| `nodeagent.image` | `node-agent-k8s` | `` |
+| `nodeagent.env.CA_PROVIDER` | `""` | `` |
+| `nodeagent.env.CA_ADDR` | `""` | `` |
+| `nodeagent.env.Plugins` | `""` | `` |
+| `nodeagent.nodeSelector` | `{}` | `` |
+
+## `pilot` options
+
+| New Key | New Default Value | New Description|
+| --- | --- | --- |
+| `pilot.autoscaleEnabled` | `true` | `` |
+| `pilot.env.PILOT_PUSH_THROTTLE_COUNT` | `100` | `` |
+| `pilot.env.GODEBUG` | `gctrace=1` | `` |
+| `pilot.cpu.targetAverageUtilization` | `80` | `` |
+| `pilot.nodeSelector` | `{}` | `` |
+| `pilot.keepaliveMaxServerConnectionAge` | `30m` | `` |
+
+
+# Old configuration options that have been removed from the current release
+
+## `ingress` options
```

Comment:
The headings for each option in this section should let readers know that the key/value pairs are no longer available to them. 
```suggestion
### Removed `ingress` key/value pairs
```